### DISCUSSION TITLE
D3W300-ZB-B AlingConel

### DISCUSSION THREAD

**Discussion by stefancvetkovic** [Koenkk/zigbee2mqtt#10380] [Devices]
Hello, I have zigbee device from Serbian company Aling Conel. 
They are quire popular in Serbia and on the Balkans.
Here is their manual for dimmer switch device under model D3W300-ZB-B: https://www.obloliving.com/manuals/user-manuals/OBLO_Living-Smart_Dimmer_D3W300ZBB-User_Manual-20170607.pdf

I am trying to put it into work with HomeAssistant with some custom extension.js
https://pastebin.com/FW8Wguu2
also I tried with `extend.light_onoff_brightness()` but the result is the same: when I doing from HA dashboard it toggles and dims correctly, but If I turn on or dimm the light from a physical device, I do not receive any updates on my HA,
What can be the issue? How can I fix that?

**Comment by kompish** [comment]
Did you managed to get it work?

**Reply by stefancvetkovic** [reply]
No, still no luck :)
But I started to build own dimmer for alingconel mask

**Reply by nljuba** [reply]
Any progress with that? Currently this device costs 71eur in Serbia, which makes it probably the most expensive Zigbee smart dimmer on the market :) and with no integration with home assisstant

**Comment by mfin** [comment]
Would love to have support for these! Just found out they use the same mounting "standard" as our popular local brand, TEM Catez, so they're high on my list for a retrofit.

**Comment by alexandar87** [comment]
@stefancvetkovic ja imam Oblo sistem kuci i par ovih uredjaja. Jesi mozda uspeo da dodas jos neki uredjaj koji nisi kupio od Oblo-a? Ja sam hteo da porucim par Zigbee senzora za vrata i detektor dima, ali nisam siguran da li se mogu upariti sa Oblo Gateway-em.
